lib/srv/desktop/rdp/rdpclient/build.rs

```diff
@@ -14,11 +14,21 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
+use std::time::SystemTime;
+
 fn main() {
+    let timestamp = SystemTime::now()
+        .duration_since(SystemTime::UNIX_EPOCH)
+        .unwrap()
+        .as_secs();
```

Comment:
nit: I can imagine a computer fast enough to build teleport in a second, but not in a nanosecond

```suggestion
        .as_nanos();
```